common/supplemental/supplementalData.xml

```diff
@@ -1979,7 +1979,7 @@ XXX Code for transations where no currency is involved
 		<language type="mzn" scripts="Arab"/>
 		<language type="mzn" territories="IR" alt="secondary"/>
 		<language type="na" scripts="Latn" territories="NR"/>
-		<language type="nan" scripts="Hans"/>
+		<language type="nan" scripts="Hans" territories="TW"/>
```

Comment:
It seems just slightly odd that these use Simplified in TW.

Reply:
It is odd -- its automatically picked up from `ConvertLanguageData` though, hmm...